doc/classes/PhysicsServer3D.xml

```diff
@@ -429,6 +430,30 @@
 				Creates a physics body. The first parameter can be any value from [enum BodyMode] constants, for the type of body created. Additionally, the body can be created in sleeping state to save processing time.
 			</description>
 		</method>
+		<method name="body_enable_continuous_collision_detection">
```

Comment:
This method could be `body_set_continuous_collision_detection` since it can be enabled or disabled.